doc/dev/cephfs-mirroring.rst

```diff
@@ -170,6 +179,29 @@ disallowed to be added for mirorring::
 Commands to check directory mapping (to mirror daemons) and directory distribution are
 detailed in `Mirror Daemon Status` section.
 
+Bootstrap Peers
+---------------
+
+Adding a peer (via `peer_add`) requires the peer cluster configuration and user keyring
+to be available in the primary cluster (manager host and hosts running the mirror daemon).
+This can be avoided by bootstrapping and importing a peer token. Peer bootstrap involves
+creating a bootstrap token on the peer cluster via::
+
+  $ ceph fs snapshot mirror peer_bootstrap create <fs_name> <client_entity> <site-name>
```

Comment:
Can you have one `client_entity` for multiple file systems on the the remote site?

Docs could be more clear about the meaning of `<site-name>`. What is it used for?

Reply:
> Can you have one `client_entity` for multiple file systems on the the remote site?

The bootstrap create command creates user via `fs authorize` command and requires a file system name. One can create a user via `fs authorize \*` and use `peer_add` to reuse one user for multiple file systems.

So, `peer_bootstrap import` kind of enforces 1:1 map between a user and a filesystem.

> 
> Docs could be more clear about the meaning of `<site-name>`. What is it used for?

Sure. I'll update the docs.